rift/rift.py

```diff
@@ -57,10 +57,13 @@ def check(m):
             channel = channels[int(choice.content)]
         else:
             channel = channels[0]
-
-        rift = OpenRift(source=author_channel, destination=channel)
-
-        self.open_rifts[author] = rift
+        
+        if author in self.open_rifts:
+            await self.bot.say("You already have a rift opened!")
+            return
+        
+        self.open_rifts[author] = OpenRift(source=author_channel, destination=channel)
```

Comment:
```suggestion
        self.open_rifts[author.id + channel.id] = OpenRift(source=author_channel, destination=channel)
```
This will allow multiple rifts for the same user. Of course this change needs to be applied in each instance in which `self.open_rifts` is used.